src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4179,7 +4179,7 @@ bool tryAsMemberOfSingleType(NamedTypeSymbol singleType, out Symbol result)
 
         private static Symbol AsMemberOfTupleType(TupleTypeSymbol tupleType, Symbol symbol)
         {
-            if (symbol.ContainingType.Equals(tupleType))
+            if (symbol.ContainingType.Equals(tupleType, SymbolEqualityComparer.ConsiderEverything))
```

Comment:
>SymbolEqualityComparer.ConsiderEverything [](start = 56, length = 41)

Is this necessary?

Reply:
Yes

---
In reply to: [305131895](https://github.com/dotnet/roslyn/pull/37247#discussion_r305131895) [](ancestors = 305131895)